gotowce/lovelace/wykres-roznych-stanow-jednego-sensora.md

```diff
@@ -11,8 +11,8 @@ author: "anybody84"
 
 ## Wygląd wykresu
 
-![Wykres trybu pracy pieca](../../../assets/images/wykres-pracy-pieca-dark.jpg)
-![Wykres trybu pracy pieca](../../../assets/images/wykres-pracy-pieca-light.JPG)
+![Wykres trybu pracy pieca](../../assets/images/wykres-pracy-pieca-dark.jpg)
```

Comment:
Może trzymać assety w tym samym miejscu, co wpis?

Reply:
Myślałem o tym, ale opierałem się na dokumentacji, gdzie pliki statyczne są trzymane w jednym worku w assetach
https://jekyllrb.com/docs/static-files/

Reply:
Dodatkowo w IDE łatwiej się pracuje gdy pliki mają różne nazwy, a nie, że każdy to index.md

Reply:
Wyyyydaje mi się, że ludziom łatwiej będzie np. dodawać gotowce, jeśli obrazy będą w tym samym folderze (albo w podfolderze assets), więc nie trzeba będzie kombinować z chodzeniem po folderach